packages/decoupled/src/site/site.ts

```diff
@@ -86,6 +88,27 @@ export class Site {
         return defaultBundleManager.getBundle(this.id)
     }
 
+    public loadConfig(): Config {
+        const configProvider = new ConfigProvider(this.id);
+        const config = configProvider.load();
+        const { configs } = config;
+
+        const start = process.hrtime();
+        this.logger.debug('Start config validation.');
+
+        const [errors, valid] = configProvider.validate(configs);
+        const end = process.hrtime(start);
+
+        this.logger.debug(`Config validation finished, took ${end[0]}.${end[1]}s`);
+
+        if (!valid) {
+            this.logger.error(errors);
+            throw new Error('Config validation failed.');
+        }
+
+        return config;
```

Comment:
How do we prevent the app from accession non registered keys? There was a requirement, that `Config.get` and `Config.set` should check if a key is registered